src/history/hs_rec.c

```diff
@@ -231,7 +231,8 @@ __hs_next_upd_full_value(WT_SESSION_IMPL *session, WT_MODIFY_VECTOR *modifies,
  *     Copy one set of saved updates into the database's history store table.
```

Comment:
Small nit: We can update the comment saying we return a boolean telling us if there was a successful write to history.